src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.js

```diff
@@ -169,10 +169,13 @@
     $(".page").hide();
     $("#stackpage").show();
 
-    $("#rawExceptionButton").click(function () {
+    $("#rawExceptionButton").click(function (event) {
```

Comment:
O_o. The fact that it's not jQuery is suuuuuuupppPpPPpeEerrRrr lame given how we're coding this. I officially hate `NodeCollection` but do like how it isn't jQuery (too heavy). ALrigh alright, `event.preventDefault()` and `event.stopPropagation()`.

Filed an issue to address making this code non-lamish: https://github.com/aspnet/Diagnostics/issues/205